antea/utils/map_functions.py

```diff
@@ -227,7 +227,6 @@ def load_corrections(filename, *,
     dst  = load_dst(filename, group, node)
     x, y = np.unique(dst[x_name].values), np.unique(dst[y_name].values)
     f, u = dst[f_name].values, dst[u_name].values
-    n    = dst[n_name].values
```

Comment:
I guess that `n` can be removed from the arguments of the function, too.

Reply:
From where?

Reply:
You mean `n_name`?

Reply:
> You mean `n_name`?

Yes.

Reply:
Done.

I couldn't find any usage of that parameter and the tests pass in my local.